content/news/2023-10-10-sla-esg-egi/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'EGI and EuroScienceGateway sign Service-Level Agreement'
+date: "2023-10-10"
+tease: "Elevating FAIR data analytics in EOSC: SLA between ESG and EGI; IISAS to bring in additional resources"
+hide_tease: false
+subsites: [all-eu, esg]
+main_subsite: eu
+tags: [eurosciencegateway, sla, egi, service level agreement]
+authors: Sebastian Schaaf
+authors_structured:
+- github: sebastian-schaaf
+---
+
+In the Galaxy-driven EOSC project [EuroScienceGateway (ESG)](https://www.eurosciencegateway.org) one of 18 partners is [EGI - the European Grid Initiative](https://www.egi.eu/about/) is a federation of computing and storage resource providers in Europe, facilitating up a distributed infrastructure. The 29 participants  from 22 countries are united by the mission to deliver advanced computing and data analytics services for research and innovation. They provide access to more than 1.2 Exabytes of research data and 1.2 Million CPU Cores for data processing. Notably, with [CESNET](https://www.egi.eu/partner/cesnet/) (CZ), [CNRS](https://www.egi.eu/partner/cnrs/) (FR), [CYFRONET](https://www.egi.eu/partner/cyfronet/) (PL), [INFN](https://www.egi.eu/partner/infn/) (IT) and [TUBITAK ULAKBIM](https://www.egi.eu/partner/tubitak-ulakbim/) (TR), five of these partners are partners of ESG, too.
+
+The Service-Level Agreement (SLA) signed between EGI and ESG formalizes access to resources on INFN's Bari cloud and [IISAS](https://www.egi.eu/partner/iisas/)'s FedCloud, adding a Slovakian location for data processing to usegalaxy.eu's resource network. In summary, the available computing resources will be extended by 130 vCPUs, 470 GB RAM and 4 TB online storage. Beyond cloud compute and storage capacities, usegalaxy.eu's virtual organization (VO) will be supported with EGI's Workload Manager and Check-in services, allowing to monitor the allocation of resources that are used to carry out tasks in the ESG project.
```

Comment:
Not sure if we need to specify the resources in that detail.

Reply:
Me neither... removed.